source/fuzz/fuzzer_pass_add_parameters.cpp

```diff
@@ -57,50 +57,16 @@ void FuzzerPassAddParameters::Apply() {
       continue;
     }
 
-    const auto* type_inst =
-        fuzzerutil::GetFunctionType(GetIRContext(), &function);
-    assert(type_inst);
-
-    // -1 because we don't take return type into account.
-    auto num_old_parameters = type_inst->NumInOperands() - 1;
-    auto num_new_parameters =
-        GetFuzzerContext()->GetRandomNumberOfNewParameters(
-            GetNumberOfParameters(function));
-
-    std::vector<uint32_t> all_types(num_old_parameters);
-    std::vector<uint32_t> new_types(num_new_parameters);
-    std::vector<uint32_t> parameter_ids(num_new_parameters);
-    std::vector<uint32_t> constant_ids(num_new_parameters);
-
-    // Get type ids for old parameters.
-    for (uint32_t i = 0; i < num_old_parameters; ++i) {
-      // +1 since we don't take return type into account.
-      all_types[i] = type_inst->GetSingleWordInOperand(i + 1);
+    auto n = GetFuzzerContext()->GetRandomNumberOfNewParameters(
+        GetNumberOfParameters(function));
```

Comment:
Please avoid very short variable names except in the case of loop variables.  I would prefer "num_new_parameters" here.

Reply:
Fixed in 93b8e3a.